R/resolver.R

```diff
@@ -352,64 +315,98 @@ resolve_vector_l <- function(expr, vector) {
       expr = rlang::eval_tidy(expr = quo, data = NULL)
     )
 
+  item_names <- vector
+  item_count <- length(vector)
+  item_sequence <- seq_along(vector)
+
```

Comment:
Add a line that checks if `resolved` is `NULL`; if so,
1. warn that `NULL` is deprecated and they should use `TRUE` instead;
2. change the value of `resolved` to `TRUE`

Reply:
Now done!